index.html

```diff
@@ -17,5 +18,68 @@
 
     <!-- All the imges you need are in img folder -->
 
+    <nav class="header col-12">
+        <div class="col-8">
+            <ul>
+                <li>Login</li>
+                <li>Help</li>
+                <li>Blog</li>
+                <li>Store</li>
+                <li>How it Works</li>
+                <li class="active">Meet Karma</li>
+            </ul>
+        </div>
+        <div class="col-4">
+            <img src="img/karma-logo.svg">
+        </div>
+    </nav> 
+
+    <main class="container">
```

Comment:
The `container` class is empty here. You need to set some rules for it in your CSS, or import Bootstrap which already uses containers.